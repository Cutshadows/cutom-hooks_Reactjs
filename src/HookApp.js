import React from 'react';


const HookApp = () => {
    return ( 
        <div> 
            <h1>Hello</h1>
        </div>
     );
}
 
export default HookApp;