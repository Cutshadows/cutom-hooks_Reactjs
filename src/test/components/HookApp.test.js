import React from 'react';
// import '@testing-library/jest-dom';
import {shallow} from 'enzyme';

import HookApp from '../../HookApp';

describe('Prueba sobre <HookApp></HookApp>', () => {
    const wrapper=shallow(<HookApp />);

    test(' probando despliegue de wrapper de hookApp ', () => {
        expect(wrapper).toMatchSnapshot();
    })
    
})
