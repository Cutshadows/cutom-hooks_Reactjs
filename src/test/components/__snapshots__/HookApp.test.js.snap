// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba sobre <HookApp></HookApp>  probando despliegue de wrapper de hookApp  1`] = `
<div>
  <h1>
    Hello
  </h1>
</div>
`;
